lib/recurly.js

```diff
@@ -93,6 +93,11 @@ const DEFAULTS = {
     card: {
       selector: '[data-recurly=card]',
       style: {}
+    },
+    brand: {
+      selector: '[data-recurly=brand]',
+      style: {},
+      inputType: 'select',
```

Comment:
If it's just Elements we're planning to support here, then we can exclude this configuration default and the change in `hosted-fields.js`